pyproject.toml

```diff
@@ -41,9 +41,7 @@ requires = ["setuptools>=42", "wheel", "setuptools_scm>=6"]
 
 [tool.pytest.ini_options]
 explicit-only = ["geospatial"]
-markers = [
-  "geospatial: tests that require the geospatial libraries",
-]
+markers = ["geospatial: tests that require the geospatial libraries"]
```

Comment:
maybe add a marker for these file-related tests and add it to the explicit-only list?